helpers/content.go

```diff
@@ -359,6 +429,15 @@ func (c *RenderingContext) getConfig() *Blackfriday {
 	return c.Config
 }
 
+func (c *RenderingContext) getMmarkConfig() *Mmark {
+	c.configInit.Do(func() {
+		if c.MmarkConfig == nil {
+			c.MmarkConfig = NewMmark()
```

Comment:
Indeed.  I just found out that mmark has implemented EXTENSION_UNIQUE_HEADER_IDS (when detecting identical, anchors add a sequence number -1, -2 etc.) and this would probably make solving #2057 easier.  There are some really nice thought-out features that make mmark an attractive alternative to Blackfriday, though I must admit I haven't quite yet switched over to mmark myself.